releaseNotes/AzureDevOpsPushPipelineExtension.md

```diff
@@ -2,6 +2,10 @@
 Azure App Configuration Push extension for Azure DevOps pipeline can be installed from the [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push). The version information can be found in the console when the task is executed:
 ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
 
+### v3.3.0 - July, 13 2022
+* Added KVSet file content profile support.
+* Fixed a bug that caused an unexpected error "Cannot convert undefined or null to object" when pushing properties from .properties file.

```

Comment:
Is there a related GitHub issue? It will be good to include it. Otherwise, this looks good to me.

Reply:
No there is no GitHub issue, the bug was posted in marketplace